react/features/conference/middleware.js

```diff
@@ -35,6 +48,11 @@ MiddlewareRegistry.register(store => next => action => {
 
         break;
     }
+    case KICKED_OUT: {
+        store.dispatch(conferenceFailed(action.conference, KICKED_OUT_REASON));
```

Comment:
Is this really a failure? I'd say it ended, in which case you don't need to navigate since it will be done automatically. WDYT?

Reply:
Yeah, I was having hard time to decide and at some point I was leaning more towards left than failed. I need to check with @lyubomir, as only failed was mentioned in the discussion, so I wonder if there was any reason behind that.